weather-app/utils/forecast.js

```diff
@@ -0,0 +1,39 @@
+/*
+API KEY - Weatherstack
+688b742c07634c35077e590aefaa25ad
```

Comment:
WHY IS THE ACCESS KEY IN A FILE!?

Reply:
Genuine mistake. Will make a note not to use such sensitive info in a script file.
I can try and use git-filter-repo, to change commit hashes. However, I do NOT want to do it as there are only 3 files, and all 3 files contain sensitive information. This would mean a full rework of commit hashes.